### PR TITLE
Fix/lowercase

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
This is what I mean

### CODE REVIEW

File: scripts/text_to_speech_node.py [hunk 1]
```diff
@@ -155,8 +155,9 @@ def speak(self, sentence_msg):
 
     def speak_srv(self, req):
         # Check if an audio file for this sentence already exists
+        # req.sentence = req.sentence.lower
```
Comment (Timple) [inline scripts/text_to_speech_node.py]: This comment is unnecessary

File: scripts/text_to_speech_node.py [hunk 2]
```diff
@@ -155,8 +155,9 @@ def speak(self, sentence_msg):
...
         for extension in ["wav", "mp3"]:
-            potential_filename = os.path.join(os.path.expanduser(self.samples_path), req.sentence + "." + extension)
+            potential_filename = os.path.join(os.path.expanduser(self.samples_path), req.sentence.lower() + "." + extension)
```
Comment (Timple) [inline scripts/text_to_speech_node.py]: Did you rename the files already?

Comment (MatthijsBurgh) [reply scripts/text_to_speech_node.py]: yep

File: scripts/text_to_speech_node.py (hunk 1 above)
Comment (MatthijsBurgh) [reply scripts/text_to_speech_node.py]: sorry, removed now

**Review by Timple — APPROVED** [review]

**Pull request merged**